src/main/java/executor/service/model/StepTypes.java

```diff
@@ -0,0 +1,17 @@
+package executor.service.model;
+
+public enum StepTypes {
```

Comment:
In my opinion, it is better to use a Class with properties here, because this implementation speaks of an exhaustive list of commands.

Reply:
We have only three types of steps, and we don't intrest to another actions, so I think it good idea, to add enum.